src/azure-cli/azure/cli/command_modules/rdbms/custom.py

```diff
@@ -596,3 +681,79 @@ def _get_tenant_id():
     sub = profile.get_subscription()
     return sub['tenantId']
 # endregion
+
+
+# region new create experience
+def create_mysql_connection_string(server_name, host, database_name, user_name, password):
+    connection_kwargs = {
+        'host': host,
+        'dbname': database_name,
+        'username': user_name,
+        'servername': server_name,
+        'password': password if password is not None else '{password}'
+    }
+    return 'mysql {dbname} --host {host} --user {username}@{servername} --password={password}'.format(**connection_kwargs)
+
+
+def create_database(cmd, resource_group_name, server_name, database_name, engine_name):
+    if engine_name == 'mysql':
+        # check for existing database, create if not present
+        database_client = cf_mysql_db(cmd.cli_ctx, None)
+    elif engine_name == 'mariadb':
+        database_client = cf_mariadb_db(cmd.cli_ctx, None)
+    try:
+        database_client.get(resource_group_name, server_name, database_name).result()
+    except CloudError:
+        logger.warning('Creating %s database \'%s\'...', engine_name, database_name)
+        database_client.create_or_update(resource_group_name, server_name, database_name, 'utf8').result()
+
+
+def form_response(username, sku, location, server_id, host, version, password, connection_string, infrastructure_encryption, database_name=None, firewall_id=None):
+    output = {
+        'host': host,
+        'username': username,
+        'password': password,
+        'skuname': sku,
+        'location': location,
+        'id': server_id,
+        'version': version,
+        'connectionString': connection_string,
+    }
+    if firewall_id is not None:
+        output['firewallName'] = firewall_id
+    if database_name is not None:
+        output['databaseName'] = database_name
+    if str(infrastructure_encryption).lower() == 'enabled':
+        output['infrastructureEncryption'] = infrastructure_encryption
+    return output
+
+
+def create_postgresql_connection_string(server_name, host, user, password):
+    connection_kwargs = {
+        'user': user,
+        'host': host,
+        'servername': server_name,
+        'password': password if password is not None else '{password}'
+    }
+    return 'postgres://{user}%40{servername}:{password}@{host}/postgres?sslmode=require'.format(**connection_kwargs)
+
+
+def check_server_name_availability(check_name_client, server_name, service_name):
+    server_availability = check_name_client.execute(server_name, service_name)
+    if not server_availability.name_available:
+        raise CLIError("The server name '{}' already exists.Please re-run command with some "
+                       "other server name.".format(server_name))
+    return True
+
+
+def update_local_contexts(cmd, server_name, resource_group_name, location, user):
```

Comment:
@arrownj could you help review local context setting here?

Reply:
arrownj approved the PR. Hence, resolving this comment